src/Common/src/Interop/Windows/kernel32/Interop.Constants.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal const int WAIT_TIMEOUT = 0x00000102;
```

Comment:
You can delete `partial class WaitOption` in the other file, and replace its uses with this definition.